wordpress-mobile/WordPress-Android
# Add new Quick Start task illustrations and placeholders

This PR adds illustrations for the "Grow Your Audience" category of Quick Start tasks. It also adds temporary placeholders for tasks that are going to be removed soon.

https://user-images.githubusercontent.com/830056/105253385-1f976100-5b5e-11eb-86ca-d55a8d5ed8e6.mp4

To test:

1. Turn on the Improved My Site flag and restart the app.
1. Go to the My Site screen.
1. Notice the Quick Start Task cards.
1. Make sure all tasks have the correct illustrations and that nothing looks off.

PR submission checklist:
